### ISSUE TITLE
Multiple visual rendering bugs; need an update to the CSS to match OkCupid's new interface

### ISSUE THREAD

**Issue by aspalter** [fabacab/pat-okcupid#21]
i've tried installing both on tampermonkey (chrome) and greasemonkey (firefox) and i was only slightly able to get it installed on firefox... only able to see the questionnaire on my "visitors" page (no others, not even splash) and only then, i had to highlight the text (visitor page text and images completely viewable under questionnaire text- sadly didn't grab a screenshot) and i don't think the red box is displaying correctly- odd tables and only viewable on "visitors" and member profile pages, not on "browse" (screenshot attached)... ![screen shot 2016-01-12 at 12 46 19 pm](https://cloud.githubusercontent.com/assets/16672317/12273558/2baae0fe-b92c-11e5-8e00-37bacc912c89.jpg)

like i said, i couldn't get the questionnaire to make itself known on chrome (my preferred browser) but it claims to be enabled...

it's been more than a year since its last update and i wonder if something has gone funky with the code... both firefox and chrome are up to date and i'm trying to run v.0.5.3

**Comment by meitar** [comment]
Yeah, OkCupid updated their UI and so this needs to be updated accordingly. However, I don't want to do it because it's a thankless job and I'm pretty fucking sick and tired of doing anti-abuse work with zero community support.

If you want to send me a patch to fix this I'll happily test and deploy it, tho.

**Comment by aspalter** [comment]
aww, that's a shame :(

i just found this yesterday after a guy on OKC started threatening me after i turned him down (surprising not the guy above)... we need more vigilant people like yourself but i get it... why put forth all of that time and energy if you don't feel appreciated?

i have absolutely no clue as to how to go about patching, this is the first userscript i've ever used, but thanks for getting back to me...

sorry you've had such a crappy experience with this :(

**Comment by fabacab** [comment]
I've actually had a great experience with this tool. It's the self-styled "social justice warriors" I've had bad experienes with. Their clique of reputation-obsessed circle jerk shitwads claim to care about sexual assault survivors but in reality a survivor's needs is a far secondary concern to them than their own self-image. Fuck that noise.

I'm working on other things these days, still free software but more general purpose than tools like this.

Anyway, don't close this issue. The bug you reported is valid and needs fixing. If the community of survivor advocates really care about making useful tools like this one available to the public, then they'll have to get their heads out of their asses and pony up some material support, and stop relying on the limited physical, emotional, and other resources of a single homeless programmer.

**Comment by aspalter** [comment]
i really wish i knew how to code

**Comment by meitar** [comment]
> i really wish i knew how to code

You can learn to: https://www.codecademy.com/learn/javascript

Clearly you already have an idea of what to do with code. If you try to learn I'm sure you'll get farther than you expect faster than you realize.

And you're welcome (read: invited) to use this project as a learning tool/environment if you like.
